diff --git "a/include\\websocketpp\\endpoint.hpp" "b/include\\websocketpp\\endpoint.hpp"
index c124b1d..9ce8a62 100644
--- "a/include\\websocketpp\\endpoint.hpp"
+++ "b/include\\websocketpp\\endpoint.hpp"
@@ -109,7 +109,7 @@ public:
 
 
     /// Destructor
-    ~endpoint<connection,config>() {}
+    ~endpoint() {}
 
     #ifdef _WEBSOCKETPP_DEFAULT_DELETE_FUNCTIONS_
         // no copy constructor because endpoints are not copyable
